packages/hamclock/hamclock.pacscript

```diff
@@ -0,0 +1,42 @@
+name="hamclock"
+pkgver="3.10"
+url="https://github.com/kj7rrv/hamclock/archive/refs/tags/v${pkgver}.tar.gz"
+homepage="https://clearskyinstitute.com/ham/HamClock"
+makedepends=("gcc" "make" "pkg-config" "libxau6" "libx11-6" "libx11-dev" "libxcb1")
+breaks=("hamclock-big" "hamclock-bigger" "hamclock-huge")
+replace=("hamclock")
+pkgdesc="Clock and world map with extra features for amateur radio. (800x480 version)"
+hash="49ee666ac5c2fd9c647119bf36a5d627f7fc8b1db22b0ad7140e9e60a30554c5"
+maintainer="Roy Williams <fang64@gmail.com>"
+repology=("project: ${name}")
+
+prepare() {
+	cd "ESPHamClock"
+	# Add -PAC to version for PACSTALL
+	sed -i 's/"/-PAC"/g' version.h
+	sed -i 's/\t-PAC"/\t"/g' version.h
+
+	# Do not check for/install updates
+	sed -i "s/tft.print (F(\"You're up to date\!\"));"'/tft.print(F("Updates disabled for PACSTALL")); tft.setCursor (tft.width()\/8, tft.height()\/3+40); tft.print(F("If this build is outdated by more than a few days,")); tft.setCursor (tft.width()\/8, tft.height()\/3+80); tft.print(F("please email fang64@gmail.com.")); wdDelay(2000);/g' ESPHamClock.ino
+	sed -i 's/bool found_newer = false;/return false;bool found_newer;/g' OTAupdate.cpp
+}
+
+build() {
+	cd "ESPHamClock"
+	make -j 4 hamclock-800x480
```

Comment:
```suggestion
	make -j"${NCPU}" hamclock-800x480
```
To utilize however many cores the user has.

Reply:
Initially I was going to hesitate making this change, but apparently it does work with more threads than 4, this is what the clearskyinstitute recommended in their documentation, but since it does still compile and work at higher threadsj it seems okay to make this change. 

It's done.